trollmoves/dispatcher.py

```diff
@@ -299,7 +317,27 @@ def run(self):
                     continue
                 destinations = self.get_destinations(msg)
                 if destinations:
-                    dispatch(msg.data['uri'], destinations)
+                    success = dispatch(msg.data['uri'], destinations)
+                    if self.publisher:
+                        self._publish(msg, destinations, success)
+
+    def _publish(self, msg, destinations, success):
+        """Publish a message.
+
+        The URI is replaced with the URI on the target server.
+
+        """
+        for url, params, client in destinations:
+            if not success[client]:
+                continue
+            del params
+            info = msg.data.copy()
+            info["uri"] = urlsplit(url).path
+            topic = self.config[client].get("publish_topic", msg.subject)
```

Comment:
The example says the publish topic is required, but here you have a default ?

Reply:
Ah, you're right. I'll update the example.

Reply:
Actually I went the other way around, log an error when the topic is missing.

Reply:
Sounds good, I could create a feedback look if the original topic is used.